components/wrist.py

```diff
@@ -1,50 +1,48 @@
 import math
 import time
 
-from magicbot import feedback, tunable
-from rev import ClosedLoopSlot, SparkMax, SparkMaxConfig
-from wpilib import DigitalInput
-from wpimath.controller import ArmFeedforward
+from magicbot import feedback
+from rev import (
+    SparkMax,
+    SparkMaxConfig,
+)
+from wpilib import AnalogEncoder
+from wpimath.controller import ArmFeedforward, PIDController
 from wpimath.trajectory import TrapezoidProfile
 
-from ids import DioChannel, SparkId
+from ids import AnalogChannel, SparkId
 from utilities.functions import clamp
 
 
 class WristComponent:
+    ENCODER_ZERO_OFFSET = math.tau / 10
```

Comment:
We'll need to measure this. I doubt we'll be able to make this exactly a 10th of a rotation.